index.html

```diff
@@ -64,6 +62,39 @@ <h2>
     <p>
       TBW
     </p>
+    <p id="credential-type-examples">
```

Comment:
in general, share a lot of the same sentiment as expressed by @marcoscaceres. One suggestion would be to remove any specific references to the credential types in this PR, and than re-add them once we have more agreement in issue #85

Reply:
Reduced the list down to the only credential type we have consensus on right now: 06271a72f5fcfbef716f61d49574b0de9449d5d0. Preserved the full list in https://github.com/WICG/digital-identities/issues/85#issuecomment-1963077832.